.github/workflows/unstale.yml

```diff
@@ -0,0 +1,45 @@
+name: "rails-bot: Unstale Issues"
+
+on:
+  issues:
+    types: [edited, commented, labeled, unlabeled, assigned, unassigned, milestoned, demilestoned, reopened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  unstale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+    steps:
+    - name: Skip updates from bot
+      if: github.actor == 'github-actions'
+      run: echo "Comment from bot, skipping job." && exit 0
```

Comment:
Is this enough to stop the further steps to run? Do we have to output something and check using `if:` later?

Reply:
Good question, I thought so but let me double check.

Reply:
It seems like any actions done by the bot are not ever triggering this action, but I don't know why so still investigating.

Reply:
Ok, finally had time to look into this. It seems GH prevents creating recursive workflows when using the repositories `GITHUB_TOKEN`.

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow#triggering-a-workflow-from-a-workflow

> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN, with the exception of workflow_dispatch and repository_dispatch, will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur.